suite-native/helpers/src/splitAddressToChunks.ts

```diff
@@ -0,0 +1,4 @@
+import { pipe, S, A } from '@mobily/ts-belt';
+
+export const splitAddressToChunks = (address: string): readonly string[] =>
```

Comment:
would be nice to have tests for this one

Reply:
fixed in https://github.com/trezor/trezor-suite/commit/8af235d6e7bb7d75e4232baa31ce1ffe754a15b6